microprofile-fault-tolerance/src/test/java/org/jboss/eap/qe/microprofile/fault/tolerance/DatabaseCrashTest.java

```diff
@@ -205,6 +205,7 @@ public void dropDatabaseSchema() {
     public static void tearDown() throws Exception {
         // stop DB if still running and delete data directory
         postgresDB.stop();
+        postgresDB.remove();
```

Comment:
I think `remove()` should be called from stop() and kill() medhod...as this might be source of failures.

Reply:
The issue I see here is that the method called `stop` removes a container. That is IMHO not desired. What I have been thinking about is implementing `Closable/AutoClosable` interface to have a `close` method. Which would care about cleanup when the container is no longer needed. WDYT?

Reply:
I've found that there can be used `--rm` option in `docker run...` which causes that container is automatically removed when stopped (or killed) what about to use it and get rid of `remove();`